src/components/SideNav/SideNav.tsx

```diff
@@ -26,6 +26,28 @@ const SideNav = (props: SideNavProps): ReactElement => {
 		link.onClick();
 	};
 
+	const findLinkselected = () => {
```

Comment:
Please be careful with the camel casing
```suggestion
	const findLinkSelected = () => {
```